extract.bat

```diff
@@ -0,0 +1,24 @@
+@echo off
+echo --^> Downloading 7-zip...
+powershell -Command Invoke-WebRequest -Uri https://www.dropbox.com/s/1g9vchwrpyzoxjx/7z.zip?dl=1 -OutFile 7z.zip
```

Comment:
i am sure you could use the official downloads instead

Reply:
sadly no, to use either 7-zip you need it's .exe and .dll
they can be downloaded from the offical website yes but its on 7z format so you would need 7-zip to extract.

Reply:
can't the user just use the context menu option i pointed out? i feel like this is pointless

Reply:
probally